scripts/lib/external_config.sh

```diff
@@ -111,5 +111,8 @@ run_chamber() {
 load_external_config() {
     local service="$1"
     local prefix="${2:-}"
-    eval "$(run_chamber env "$service" | sed -E "s/(^export +)(.*)/\1${prefix}\2/")"
+    local env_output
+    env_output=$(run_chamber env "$service")
+    [[ -z "$env_output" ]] && echo "WARNING: external configuration is empty for service '$service'"
```

Comment:
The notions of "external configuration" and "service" seem to carefully avoid leaking abstractions, but I think if something goes wrong it is better to be explicit.
```suggestion
    [[ -z "$env_output" ]] && echo "WARNING: no parameters found under '/$service' in AWS parameter store of this environment"
```